cmd/cored/main.go

```diff
@@ -191,12 +187,7 @@ func main() {
 		chainlog.Fatalkv(ctx, chainlog.KeyError, errors.Wrap(err, "Serve"))
 	}()
 
-	if useTLS {
-		http.DefaultTransport.(*http.Transport).TLSClientConfig = &tls.Config{
-			Certificates: []tls.Certificate{*cert},
-			RootCAs:      loadRootCAs(*rootCAs),
-		}
-	}
+	http.DefaultTransport.(*http.Transport).TLSClientConfig = tlsConfig
```

Comment:
Yeah I think we should go in that direction, but I was hesitant to do too much all at once.

> could be a race with `raft.Start`

😮 Indeed. Will fix in the next PR.